tests/phpunit/integration/tests/Admin/Editor.php

```diff
@@ -20,8 +20,21 @@
 
 namespace Google\Web_Stories\Tests\Integration\Admin;
 
+use Google\Web_Stories\Admin\Google_Fonts;
```

Comment:
I believe there is a sniff for this here. https://github.com/slevomat/coding-standard/blob/master/doc/namespaces.md#slevomatcodingstandardnamespacesreferenceusednamesonly-

I would love to see as use import like this through the test, it makes the code way more readable. Love to add a lint at the same time / in another PR.

Reply:
Awesome, let's discuss there